src/test/app/MPToken_test.cpp

```diff
@@ -1316,6 +1316,157 @@ class MPToken_test : public beast::unit_test::suite
         }
     }
 
+    void
+    testDepositPreauth()
+    {
+        testcase("DepositPreauth");
+
+        using namespace test::jtx;
+        Account const alice("alice");  // issuer
+        Account const bob("bob");      // holder
+        Account const diana("diana");
+        Account const dpIssuer("dpIssuer");  // holder
+
+        const char credType[] = "abcde";
+
+        {
+            Env env(*this);
+
+            env.fund(XRP(50000), diana, dpIssuer);
+            env.close();
+
+            MPTTester mptAlice(env, alice, {.holders = {bob}});
+            mptAlice.create(
+                {.ownerCount = 1,
+                 .holderCount = 0,
+                 .flags = tfMPTRequireAuth | tfMPTCanTransfer});
```

Comment:
Why does this test involve `tfMPTRequireAuth`? Seems kind of unnecessary.

Reply:
didn't check MPT functionality, just copying it here